test/test__helper.rb

```diff
@@ -43,6 +43,7 @@ def load_it(judge, fb)
     $judge = judge
     $options = Judges::Options.new({ 'repositories' => 'foo/foo' })
     $loog = Loog::NULL
+    Fbe.github_graph(options: Judges::Options.new('testing' => true))
```

Comment:
@Suban05 what is this for? It looks very procedural. Why do you call this method and don't use the result returned?

Reply:
@yegor256 good question... I did this so that future calls to `Fbe.github_graph` will return `Fbe::Graph::Fake`.
For example, I want a fake object here:
`comments_resolved: Fbe.github_graph.resolved_conversations`

Reply:
@Suban05 just put `testing => true` into the `$options` two lines before this one

Reply:
@yegor256 but if I put `testing => true` into the `$options`, then `Fbe.octo` will also be fake and a lot of tests will fail

Reply:
@Suban05 then, don't do this configuration inside the `test__helper.rb`. It is used by all tests, but the behavior that you are trying to add is not needed by all of them. Use `options` where they are needed, in each particular test.

Reply:
@yegor256 thanks, fixed